app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/insurance/TypeOfContractExt.kt

```diff
@@ -63,3 +63,65 @@ fun TypeOfContract.toContractType(): ContractType = when (this) {
 
   is TypeOfContract.UNKNOWN__ -> ContractType.UNKNOWN
 }
+
+fun String.toContractType(): ContractType = when (this) {
```

Comment:
Will we need to do this locally with the new API?

Reply:
Good question, there is an enum in the schema from promise-cms actually. I can investigate if its possible to use that 👍